draft-thomson-ppm-prss.md

```diff
@@ -206,7 +206,7 @@ A randomness context is a concept that is defined by protocols that use PRSS.
 Each context is identified by a unique string of bytes.  This string is passed
 to the KDF to produce a shared value that is unique to that context.
 
-This document uses the system of describing, naming, and identifying KEMs
+This document uses the system of describing, naming, and identifying KDFs
```

Comment:
I suspect this was copy-pasted from the previous section discussing KEMs. KDFs are more relevant here right?